nf_core/pipeline-template/lib/NfcoreSchema.groovy

```diff
@@ -126,36 +133,36 @@ class NfcoreSchema {
 
         //=====================================================================//
         // Validate parameters against the schema
-        InputStream inputStream = new File(jsonSchema).newInputStream()
-        JSONObject rawSchema = new JSONObject(new JSONTokener(inputStream))
+        InputStream input_stream = new File(getSchemaPath(workflow, schema_filename=schema_filename)).newInputStream()
+        JSONObject raw_schema = new JSONObject(new JSONTokener(input_stream))
 
         // Remove anything that's in params.schema_ignore_params
-        rawSchema = removeIgnoredParams(rawSchema, params)
+        raw_schema = removeIgnoredParams(raw_schema, params)
```

Comment:
The java convention is to use camelCase instead of underscores, that's why I have been using it here. Shouldn't we stick to the standard convention instead of pythonizing the groovy code?

Reply:
You are absolutely right and I am not sure what the right thing to do here is to be honest... It was more to stick with the nf-core convention we have standardised across pipelines i.e. function names in camelCase and all parameters/variables in snake_case.